CONTRIBUTING.md

```diff
@@ -75,10 +75,12 @@ The VM doesn't include:
    [vagrant-librarian-chef](https://github.com/jimmycuadra/vagrant-librarian-chef),
    [vagrant-omnibus](https://github.com/schisamo/vagrant-omnibus),
    [vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest):
+   [vagrant-newdisk](https://github.com/dbfit/vagrant-newdisk):
 
         vagrant plugin install vagrant-librarian-chef
         vagrant plugin install vagrant-omnibus
         vagrant plugin install vagrant-vbguest
+        vagrant plugin install vagrant-newdisk
```

Comment:
Yes, I think there are certain limitations. I think max number of `ports` per IDE controller is 2. I'm not sure about devices (I guess it should be OK to add at least 10). We'll think about it if we reach the limit.
Also in case of more disks it's likely the new device not to be `/dev/sdb`. We should check that when we change the vagrant base box.